Sources/LaunchItem.swift

```diff
@@ -44,16 +45,18 @@ public enum LaunchItem: CustomStringConvertible, Equatable {
             return true
         case let (.userActivity(itemLHS), .userActivity(itemRHS)) where itemLHS == itemRHS:
             return true
+        case let (.sourceApplication(itemLHS), .sourceApplication(itemRHS)) where itemLHS == itemRHS:
+            return true
         case (.none, .none):
             return true
-        default:
+        case (.remoteNotification, _), (.localNotification, _), (.openURL, _), (.shortcut, _), (.userActivity, _), (.sourceApplication, _), (.none, _):
```

Comment:
👏

Reply:
for some reason, it seems like CI doesn't think this switch is exhaustive 🤔. I'm guessing older Swift compilers had a bug with exhaustive switch statements with cases that had associated values? 

If I'm right that this is an old Swift compiler vs new Swift compiler, our options are:
1. We can bump the Xcode version in CI to fix this, which will require us to update our min Xcode documentation
2. We can go back to using a `default` here (but let's add a comment as to why we're using a `default` statement).

Reply:
😞 Let's make it a `default` with a comment for now, and look at bumping the Xcode version number with the next pod version bump.